build/postbuild.ps1

```diff
@@ -59,8 +59,6 @@ foreach($nuspec in $nuspecs)
 				$item.Pushed = $true;
 			}
 		}
-		
-		$item
```

Comment:
Ahh, all I saw was a truncated path like below. I'll add it back with Format-List.

```
Path
----
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.MetadataReader\Microsoft.Fx.Portability.Met...
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability\Microsoft.Fx.Portability.1.4.0-alpha-00000....
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Offline\Microsoft.Fx.Portability.Offline.1....
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Reports.Html\Microsoft.Fx.Portability.Repor...
E:\git\repos\conniey\dotnet-apiport\bin\Release\Microsoft.Fx.Portability.Reports.Json\Microsoft.Fx.Portability.Repor...
```